src/sutta_publisher/shared/github_handler.py

```diff
@@ -0,0 +1,43 @@
+import json
+import logging
+import os
+from base64 import b64encode
+from io import BytesIO
+
+import requests
+
+log = logging.getLogger(__name__)
+
+
+class GitHubHandler:
+    _REPO_URL = "https://api.github.com/repos/suttacentral/editions/contents/{file_name}"
+
+    @classmethod
+    def upload_file_to_repo(cls, file_path: str, file: BytesIO) -> None:
```

Comment:
I'd suggest adding explanation to the parameters in docstring, at least for me it's not obvious at first glance why we need both `file_path` and `file`